Gosu/Graphics.hpp

```diff
@@ -69,13 +69,10 @@ namespace Gosu
         static void transform(const Transform& transform,
                               const std::function<void ()>& f);
 
-        //! Draws a line from one point to another (last pixel exclusive).
-        //! Note: OpenGL lines are not reliable at all and may have a missing pixel at the start
-        //! or end point. Please only use this for debugging purposes. Otherwise, use a quad or
-        //! image to simulate lines, or contribute a better draw_line to Gosu.
         static void draw_line(double x1, double y1, Color c1,
                               double x2, double y2, Color c2,
-                              ZPos z, AlphaMode mode = AM_DEFAULT);
+                              ZPos z, AlphaMode mode = AM_DEFAULT,
+                              double thickness=1.0);
```

Comment:
+space around `=`

Reply:
I'm not a fan of adding a parameter at the end :/ But I also don't see any sane way to avoid this without subtly breaking code.

Reply:
I would prefer it right before z as well but no idea how to not break code too. I mean in Ruby we could use a named parameter but a list of 8(!) normal and 1 named param would look a bit akward I think. But thickness seems like a good Addition to use draw line especially once someone codes even fancier lines 😛

Reply:
Yeah, the 8 parameter thing is another good theme for post-1.0! :/